app/views/posts/index.html.erb

```diff
@@ -1,8 +1,24 @@
-<h1 style='text-align:center'>Post list</h1>
+<div style="margin-top: 10px">
+ <% if @user_search.nil? %>
+<h4>User not found</h4>
+<% else %>
+<div>
+<%= render partial: 'users/user', locals: { user: @user_search } %>
+<div>
+<% end %>
+</div>
+<div>
 <%= @posts.each_with_index do |post, index| %>
- <div style="border: 2px solid black; margin: 5px">
- <h3>post <%= index + 1 %>
-    <p><%= post.title %></p>
+  <div class="container border rounded" style="margin-top: 5px">
+    <h4>Post #<%= index + 1 %></h4>
     <p><%= post.text %></p>
- </div>
-<% end %>
```

Comment:
- [ ] Kindly create a link that leads to the **_Post Show Page_** when a user is on the **_Post Index page_**. This will improve the user experience and make it easy to navigate between pages. 😎 💻